ietf/draft-ietf-jmap-calendars-18.txt

```diff
@@ -0,0 +1,3360 @@
+               "sendTo":{
+                 "imip":"mailto:joe@example.com"
+               },
+               "kind":"individual",
+               "roles":{
+                 "attendee":true
+               },
+               "participationStatus":"needs-action",
+               "expectReply":true
+             }
+           },
+           "mayInviteSelf":false,
+           "mayInviteOthers":false,
+           "useDefaultAlerts":false,
+           "alerts":null
+         }
+       },
+       "sendSchedulingMessages":true
+
+
+
+Jenkins & Douglass       Expires 9 December 2024               [Page 44]
+
+Internet-Draft               JMAP Calendars                    June 2024
+
+
+     }, "0"]
+   ]
+
+   As the event has participants, the server sets a "replyTo" property.
+   This server uses a special email address for receiving iMIP RSVPs
+   ([RFC5546]) rather than just receiving them at the owner's regular
+   email address, and also provides a web page for people that don't
+   have calendar clients supporting iMIP.  The response may look
+   something like this:
+
+   NOTE: '\' line wrapping per RFC 8792
+
+   [
+     ["CalendarEvent/set", {
+       "accountId":"a0x9",
+       "created":{
+         "k559":{
+           "id":"E-5d5776f6-ff8e-4bfd-ab3e-fe2fe5d4fa91",
+           "isOrigin": true,
+           "@type": "Event",
+           "created": "20221005T20:42:13Z",
+           "updated": "20221005T20:42:13Z",
+           "sequence": 1,
+           "replyTo": {
+             "imip": "3e87-1b18bb5e6b4@itip.example.com",
+             "web": "https://cal.example.com/\
+                   5d5776f6-ff8e-4bfd-ab3e-fe2fe5d4fa91/?\
+                   auth=bfc0-4ba3-9e44"
+           }
+         }
+       },
+       ...
+     }, "0"]
+   ]
+
+8.3.  Snoozing an alert
+
+   The client is connected to the event source and receives a push:
+
+   {
+           "@type": "CalendarAlert",
+           "accountId": "a0x9",
+           "calendarEventId": "E-7e93e3ee-4e6e-408a-9adc-cbaf1dbd0a3f",
+           "uid": "b6f7e27b-5872-4b52-b457-0242541bb01c",
+           "recurrenceId": null,
+           "alertId": "7519a951-1e6f-4a6c-b08b-20dd2e5a89cd"
+   }
+
+
+
+
+Jenkins & Douglass       Expires 9 December 2024               [Page 45]
+
+Internet-Draft               JMAP Calendars                    June 2024
+
+
+   Not finding this event in its local cache, the client fetches the
+   information for this event that it needs to show the alert by making
+   the following request:
+
+   [
+     ["CalendarEvent/get", {
+           "accountId": "a0x9",
+           "ids":["E-7e93e3ee-4e6e-408a-9adc-cbaf1dbd0a3f"],
+           "properties": ["calendarIds", "title", "start",
+             "timeZone", "useDefaultAlerts", "alerts"]
+     }, "0"]
+   ]
+
+   In response it receives:
+
+   [
+     ["CalendarEvent/get", {
+           "accountId": "a0x9",
+           "list": [{
+          "id": "E-01c9626e-1490-43df-a34f-457021256281",
```

Comment:
This id in the response does not match the `ids` list in the get above, should it?

Reply:
Thanks, fixed.